ConsentViewController/Classes/SourcePointClient/SPNativeUI.swift

```diff
@@ -164,15 +164,17 @@ class SPNativeButton: SPNativeUI {
 class SPNativeImage: SPNativeUI {
     class Settings: SPNativeUISettings {
         let src: String
-
+        let url: URL?
```

Comment:
What's the difference between `src` and `url`? Does the message json contain both?

Reply:
Yup it does. src always leads to default logo and url leads to logo specified in template builder